tests/combobox-select-only/data/tests.csv

```diff
@@ -1,33 +1,33 @@
+1,Navigate forwards to a collapsed select-only combobox in reading mode,"JAWS,NVDA",reading,navigate forwards to collapsed select-only combobox,moveFocusBeforeCombobox,sets focus on a link before the Combobox,combobox aria-labelledby aria-expanded,With the reading cursor on the 'Navigate forwards from here' link, navigate to the 'Favorite Fruit' Combobox.,Role 'combobox' is conveyed,Name 'Favorite Fruit' is conveyed,Text of the selected option ('Choose a Fruit') is conveyed,State of the combobox (collapsed) is conveyed,,,
+2,Navigate backwards to a collapsed select-only combobox in reading mode,"JAWS,NVDA",reading,navigate backwards to collapsed select-only combobox,moveFocusAfterCombobox,sets focus on a link after the Combobox,combobox aria-labelledby aria-expanded,With the reading cursor on the 'Navigate backwards from here' link, navigate to the 'Favorite Fruit' Combobox.,Role 'combobox' is conveyed,Name 'Favorite Fruit' is conveyed,Text of the selected option ('Choose a Fruit') is conveyed,State of the combobox (collapsed) is conveyed,,,
+3,Navigate forwards to a collapsed select-only combobox in interaction mode,"JAWS,NVDA",interaction,navigate forwards to collapsed select-only combobox,moveFocusBeforeCombobox,sets focus on a link before the Combobox,combobox aria-labelledby aria-expanded,With focus on the 'Navigate forwards from here' link, navigate to the 'Favorite Fruit' Combobox.,Role 'combobox' is conveyed,Name 'Favorite Fruit' is conveyed,Text of the selected option ('Choose a Fruit') is conveyed,State of the combobox (collapsed) is conveyed,,,
+4,Navigate backwards to a collapsed select-only combobox in interaction mode,"JAWS,NVDA",interaction,navigate backwards to collapsed select-only combobox,moveFocusAfterCombobox,sets focus on a link after the Combobox,combobox aria-labelledby aria-expanded,With focus on the 'Navigate backwards from here' link, navigate to the 'Favorite Fruit' Combobox.,Role 'combobox' is conveyed,Name 'Favorite Fruit' is conveyed,Text of the selected option ('Choose a Fruit') is conveyed,State of the combobox (collapsed) is conveyed,,,
+5,Navigate forwards to a collapsed select-only combobox,voiceover_macos,interaction,navigate forwards to collapsed select-only combobox,moveFocusBeforeCombobox,sets focus on a link before the Combobox,combobox aria-labelledby aria-expanded,With focus on the 'Navigate forwards from here' link, navigate to the 'Favorite Fruit' Combobox.,Role 'combobox' is conveyed,Name 'Favorite Fruit' is conveyed,Text of the selected option ('Choose a Fruit') is conveyed,State of the combobox (collapsed) is conveyed,,,
```

Comment:
`With focus on the 'Navigate forwards from here' link, navigate to the 'Favorite Fruit' Combobox.` should be wrapped in double quotes due to mid sentence comma.